kamelets/salesforce-delete-sink.kamelet.yaml

```diff
@@ -0,0 +1,116 @@
+    camel.apache.org/provider: "Apache Software Foundation"
+    camel.apache.org/kamelet.group: "Salesforce"
+  labels:
+    camel.apache.org/kamelet.type: "sink"
+spec:
+  definition:
+    title: "Salesforce Delete Sink"
+    description: |-
+      Removes an object from Salesforce. The body received must be a json
+      containing two keys: sObjectId and sObjectName.
+
+      Example body: { "sObjectId": "XXXXX0", "sObjectName": "Contact" }
+    required:
+      - clientId
+      - clientSecret
+      - userName
+      - password
+    type: object
+    properties:
+      loginUrl:
+        title: Login URL
+        description: The Salesforce instance login URL
+        type: string
+        default: https://login.salesforce.com
+      clientId:
+        title: Consumer Key
+        description: The Salesforce application consumer key
+        type: string
+        x-descriptors:
+        - urn:camel:group:credentials
+      clientSecret:
+        title: Consumer Secret
+        description: The Salesforce application consumer secret
+        type: string
+        format: password
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:password
+        - urn:camel:group:credentials
+      userName:
+        title: Username
+        description: The Salesforce username
+        type: string
+        x-descriptors:
+        - urn:camel:group:credentials
+      password:
+        title: Password
+        description: The Salesforce user password
+        type: string
+        format: password
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:password
+        - urn:camel:group:credentials
+  types:
+    in:
+      mediaType: application/json
+      schema:
+        type: object
+        properties:
```

Comment:
if any of the property is mandatory, it should be listed among the required fields